com.rioterm.Rio.metainfo.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.rioterm.Rio</id>
+  <content_rating type="oars-1.1"/>
+
+  <name>Rio</name>
+  <summary>Hardware-accelerated GPU terminal emulator powered by WebGPU</summary>
+
+  <description>
+    <p>Rio is a hardware-accelerated GPU terminal emulator. It is designed to be fast and
+      efficient, with a focus on performance and usability.</p>
+    <p>Rio is built using the latest technologies, including Rust and WebGPU, to provide a
+      modern and responsive user experience. It supports a wide range of features, including
+      split panes, tabs, and customizable themes.</p>
+    <p>Rio is open source and available for Windows, macOS, and Linux.</p>
+  </description>
+
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+
+  <launchable type="desktop-id">com.rioterm.Rio.desktop</launchable>
+
+  <url type="homepage">https://rioterm.com/</url>
+  <url type="help">https://rioterm.com/docs/config</url>
+  <url type="bugtracker">https://github.com/raphamorim/rio/issues</url>
+  <url type="donation">https://github.com/sponsors/raphamorim</url>
+
+  <developer id="io.raphamorim">
+    <name>Raphael Amorim</name>
+    <url type="homepage">https://raphamorim.io/</url>
+  </developer>
+
+  <recommends>
+    <control>keyboard</control>
+    <control>pointing</control>
+    <control>touch</control>
+  </recommends>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>Split panels</caption>
+      <image>https://rioterm.com/assets/images/demo-split-2371bfcbad15af3a7c820db4811c1032.png</image>
```

Comment:
I think the quality of the screenshots here is subpar, I'll ask the maintainer to add a few better images to the official docs and use them here instead

Reply:
I will add these screenshots to the metainfo and move it to the upstream repo, meanwhile are the quality of these screenshots acceptable?

![default](https://github.com/user-attachments/assets/1d392f78-4eff-48b6-84a9-3c8e3fd883b1)

![ligatures](https://github.com/user-attachments/assets/6d985b09-36a6-42ef-bdc7-553068d011fa)

![sixel](https://github.com/user-attachments/assets/c42d2ff8-d97e-4a8f-8388-00cb0f06ce9a)

![split-panels](https://github.com/user-attachments/assets/dadf923a-5768-4a6f-ad8d-94a2dd4267da)

![transparency](https://github.com/user-attachments/assets/7c158c6f-90b6-4126-b919-08f59a9cc0d5)

Reply:
yea it's fine